be/src/storage/storage_engine.h

```diff
@@ -378,6 +386,14 @@ class StorageEngine {
 
     HeartbeatFlags* _heartbeat_flags = nullptr;
 
+    std::unordered_map<int64_t, std::pair<int64_t, int64_t>*> _tableid_auto_increment_map;
```

Comment:
use unique_ptr?

Reply:
maybe we can use shared_ptr? If use shared_ptr, we can just use a simple mutex to protect the map struct and ignore the value (auto increment meta info).